README.rst

```diff
@@ -283,8 +283,7 @@ How to Contribute
 -----------------
 
 Contributions are very welcome. The easiest way is to fork this repo, and then
-make a pull request from your fork. The first time you make a pull request, you
-may be asked to sign a Contributor Agreement.
```

Comment:
lol, this was from the old days when Edx managed this project. I cant believe I never noticed.

Reply:
Interesting history :-)